src/main/java/com/scottlogic/librarygradproject/Services/BorrowService.java

```diff
@@ -73,27 +73,19 @@ public void bookReturned(long borrowId) {
             throw new BookAlreadyReturnedException(borrowId);
         }
         borrowToReturn.setActive(false);
+        borrowToReturn.setReturnDate(LocalDate.now());
```

Comment:
I assume this is for Librians?

Reply:
When we have the loan history on the profile page it makes more sense for the last column to show "date returned" rather than the original arbitrary due date so we're setting it on return